docs/dev/test-configuration-options.md

```diff
@@ -579,16 +601,63 @@ Appium override setting that enables the interception of biometric input, allowi
 
 ---
 ### `groupFolderRedirectEnabled`
-<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span></small> | <small><span className="sauceDBlue">iOS Only</span> | </small></p>
+<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> | <span className="sauceDBlue">iOS Only</span> | </small></p>
 
 Appium override setting that enables the use of the app's private app container directory instead of the shared app group container directory. For testing on the Real Device Cloud, the app gets resigned, which is why the shared directory is not accessible.
 
 ---
 ### `systemAlertsDelayEnabled`
-<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span></small> | <small><span className="sauceDBlue">iOS Only</span> | </small></p>
+<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> | <span className="sauceDBlue">iOS Only</span> | </small></p>
 
 Appium override setting that delays system alerts, such as alerts asking for permission to access the camera, to prevent app crashes at startup.
 
+---
+### `tunnelName`
```

Comment:
I don't think we should add `tunnelName` and `tunnelOwner` to our docs until we have selenium implementation in order to avoid any possible confusion

Reply:
Hey @waggledans parentTunnel and tunnelIdentifier were already confusing everyone, including Saucers. :)

I'd like to go ahead with reducing this confusion for RDC customers by adding the new nomenclature whilst making it clear that it's only for RDC.